include/deal.II/matrix_free/portable_matrix_free.h

```diff
@@ -73,7 +73,7 @@ namespace Portable
    * of blocks in a BlockVector and the number of FEEvaluation objects that can
    * be active in a single cell_loop().
    */
-  static constexpr const unsigned int n_max_dof_handlers = 5;
+  inline constexpr const unsigned int n_max_dof_handlers = 5;
```

Comment:
While you're there, the `const` is redundant with `constexpr`
```suggestion
  inline constexpr unsigned int n_max_dof_handlers = 5;
```
We use `constexpr const` for variables in other places as well, though.

Reply:
Ah, hm. 

Can I defer that for the current patch and just follow up with a patch that fixes it everywhere?

Reply:
Of course; I didn't mean to block on the comment.